commands/channel_test.go

```diff
@@ -133,6 +136,277 @@ func (s *MmctlUnitTestSuite) TestSearchChannelCmdF() {
 	})
 }
 
+func (s *MmctlUnitTestSuite) TestArchiveChannelCmdF() {
+	s.Run("Archive channel without args returns an error", func() {
+		printer.Clean()
+
+		err := archiveChannelsCmdF(s.client, &cobra.Command{}, []string{})
+		mockErr := errors.New("Enter at least one channel to archive")
+
+		expected := mockErr.Error()
+		actual := err.Error()
+		 
+		s.Require().Equal(expected, actual)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
+
+	s.Run("Archive an existing channel on an existing team", func() {
+		printer.Clean()
+
+		teamArg := "some-team-id"
+		mockTeam := model.Team{Id: teamArg}
+		channelArg := "some-channel"
+		channelID := "some-channel-id"
+		mockChannel := model.Channel{Id: channelID, Name: channelArg}
+
+		cmd := &cobra.Command{}
+		args := fmt.Sprintf("%s:%s", teamArg, channelArg)
```

Comment:
To simplify the code here, we can directly use:

```suggestion
		args := teamArg + ":" + channelArg
```